bazel/external/wamr.BUILD

```diff
@@ -0,0 +1,24 @@
+licenses(["notice"])  # Apache 2
+
+load("@rules_foreign_cc//foreign_cc:defs.bzl", "cmake")
+
+package(default_visibility = ["//visibility:public"])
+
+filegroup(
+    name = "srcs",
+    srcs = glob(["**"]),
+    visibility = ["//visibility:public"],
+)
+
+cmake(
+    name = "libiwasm",
+    cache_entries = {
+        "CMAKE_BUILD_TYPE": "Debug",
```

Comment:
I think you should remove this, Bazel is going to automatically set this to `Debug` when using `-c dbg`, and `Release` when using `-c fastbuild` and `-c opt`.

Reply:
Fixed